pkg/sip/outbound.go

```diff
@@ -330,14 +340,20 @@ func (c *outboundCall) setStatus(v CallStatus) {
 		return
 	}
 	r.LocalParticipant.SetAttributes(map[string]string{
-		AttrSIPCallStatus: attr,
+		livekit.AttrSIPCallStatus: attr,
 	})
 }
 
 func (c *outboundCall) sipSignal(ctx context.Context) error {
 	ctx, span := tracer.Start(ctx, "outboundCall.sipSignal")
 	defer span.End()
 
+	if c.sipConf.ringingTimeout > 0 {
```

Comment:
good for safety, but it seems that this is impossible given the defaults that we are setting.

Reply:
I'd prefer to keep it for safety, if you don't mind. In case the timeout is removed later, this won't break at least.

Reply:
yup for sure!